### PR TITLE
Merge of develop into feature/grpc

### PR DESCRIPTION
**Pull request by chburket**

### CODE REVIEW

File: docs/README-develop-beaker-scripts.md [hunk 1]
```diff
@@ -61,6 +67,8 @@ configure terminal
 end
 ~~~
 
+*Note: To enable sshd inside the `open agent container (OAC)` reference `OAC` documentation.
```
Comment (glennmatthews) [inline docs/README-develop-beaker-scripts.md]: Looks like there's a missing * at the end of this line (inherited from develop, but it's messing up the XR section).

**Comment by glennmatthews** [comment]
Possibly relevant comparison link (showing just how this branch differs from develop, i.e. the XR-specific content that will be in feature/grpc after the merge):

https://github.com/cisco/cisco-network-puppet-module/compare/develop...chburket:feature/grpc

**Comment by chrisvanheuveln** [comment]
:+1: Per our earlier agreement to merge as is and pick up the pieces in develop. 

Please fix the rubocop failure though...

**Comment by glennmatthews** [comment]
Just to be clear - this PR is for merging the latest develop into feature/grpc - collapsing feature/grpc into develop will be the next step. :smiley:

**Comment by chrisvanheuveln** [comment]
sigh... just anxious to get rid of this branch...

File: examples/cisco/demo_bgp.pp [hunk 2]
```diff
@@ -14,12 +14,13 @@
...
 # limitations under the License.
 
-class ciscopuppet::demo_bgp {
+class ciscopuppet::cisco::demo_bgp {
 
   # --------------------------------------------------------------------------#
   # Configure Global BGP                                                      #
   # --------------------------------------------------------------------------#
 
+<<<<<<< HEAD:examples/demo_bgp.pp
```
Comment (glennmatthews) [inline examples/cisco/demo_bgp.pp]: Please fix merge conflicts in this file.

**Comment by chburket** [comment]
Fixed issues.

**Pull request merged**